NOTICE

```diff
@@ -43,3 +43,22 @@ The Apache Flink project contains or reuses code that is licensed under the Apac
 - Google Cloud Client Library for Java (https://github.com/googleapis/google-cloud-java) Copyright 2017 Google LLC
 
   See: flink-end-to-end-tests/flink-connector-gcp-pubsub-emulator-tests/src/test/java/org/apache/flink/streaming/connectors/gcp/pubsub/emulator/PubsubHelper.java
+
+- aws-sdk-java-s3 (https://github.com/aws/aws-sdk-java)
+
+  See: flink/flink-filesystems/flink-s3-fs-base/src/main/java/com/amazonaws/services/s3/model/transform/XmlResponsesSaxParser.java
+
+AWS SDK for Java
+Copyright 2010-2014 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+This product includes software developed by
+Amazon Technologies, Inc (http://www.amazon.com/).
+
+**********************
+THIRD PARTY COMPONENTS
+**********************
```

Comment:
I was puzzled where you get this and it's just a copy paste from the library's notice file 👍 

https://github.com/aws/aws-sdk-java/blob/master/NOTICE.txt